main.go

```diff
@@ -39,8 +40,8 @@ var (
 
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
-
 	utilruntime.Must(idmv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(routev1.AddToScheme(scheme))
```

Comment:
INFO Required for using OpenShift objects; This add the objects API to the scheme.